src/react-apps/ux-editor/src/sagas/formDesigner/formDesignerSagas.ts

```diff
@@ -264,14 +265,33 @@ export function* watchUpdateDataModelBindingSaga(): SagaIterator {
 
 function* updateFormComponentSaga({
   updatedComponent,
-  id
+  id,
+  activeContainer,
 }: FormDesignerActions.IUpdateFormComponentAction): SagaIterator {
+  const formDesignerState: IFormDesignerState = yield select(selectFormDesigner);
+  const baseContainerId = Object.keys(formDesignerState.layout.order)[0];
   try {
     yield call(
       FormDesignerActionDispatchers.updateFormComponentFulfilled,
       updatedComponent,
       id
     );
+    // update Container with dataModelBinding
+    console.log(activeContainer, baseContainerId);
+    if (activeContainer !== baseContainerId) {
+      const tempContainer = formDesignerState.layout.containers[activeContainer];
+      const array = updatedComponent.dataModelBinding.split('.');
+      const length = (array[0].length + array[1].length + 1);
+      const string = updatedComponent.dataModelBinding.substr(0, length);
+      tempContainer.dataModelGroup = string;
+
+      yield call(
+        FormDesignerActionDispatchers.updateFormContainerFulfilled,
+        tempContainer,
+        activeContainer
+      );
+    }
+
```

Comment:
Is this how the dataModelGroup is set when a component is added in it? It's not a general rule that the data model group is the first two "items" in the data model, that is just the specific example we have been using for testing.

I suggest we remove this for now, and consider adding this functionality in connection with #218